aemedge/blocks/hero/hero.js

```diff
@@ -1,6 +1,25 @@
 // import { createOptimizedPicture } from '../../scripts/aem.js';
 import { createTag } from '../../scripts/utils.js';
 
+function getVideoUrl(videoLinks) {
+  const screenWidth = window.innerWidth;
+
+  if (screenWidth >= 1024) {
+    // Desktop view (1024px and above)
+    return videoLinks.find((link) => link.getAttribute('href').includes('desktop'))?.getAttribute('href');
```

Comment:
It was a good idea, but I think the naming convention is not standardized. You will have to select the video based on the order. 

The blog pages have the desktop image 1st, and mobile image last, so let's use the same order for the videos. 
- 1st video link is destop, 
- 2nd is tablet, and 
- 3rd is mobile. 
- If there are only 2 videos, the use the 1st video for both desktop and tablet, and the 2nd one for mobile.

Current sling.com homepage video names for example:
<img width="928" alt="image" src="https://github.com/user-attachments/assets/2fcb58da-73e2-44e9-9c64-56eddf42fb63">